week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -18,24 +18,38 @@
 
 let array = [12, 73, 92, 45, 100, 14, 61];
 
-array.some((value) => {return (value % 2 == 0)}); /* this will return true as SOME values 
+array.some((value) => {
+  return value % 2 == 0;
+}); /* this will return true as SOME values 
 will have a remainder of 0 i.e. they are even numbers*/
 
-array.every((value) => {return (value % 2 == 0)}); /* this will return false as not ALL 
+array.every((value) => {
+  return value % 2 == 0;
+}); /* this will return false as not ALL 
 values will have a remainder of 0 i.e. there are some odd numbers in the array too*/
 
 /* + + + + + + + + + + + + + + */
 
 /* Now try to complete the exercise */
+// let bushBerryColours1 = ["pink", "pink", "pink", "neon", "pink", "transparent"];
 
-function bushChecker() {
+function bushChecker(color) {
+  let result = color.every((value) => {
+    return value === "pink";
+  });
+  console.log(result);
+  // console.log(bushBerryColours1.every(bushChecker));
 
+  if (result === true) {
+    return "Bush is safe to eat from";
+  } else {
+    return "Toxic! Leave bush alone!";
+  }
```

Comment:
There is a shorter way of writing this function, see the solution here https://github.com/anthonytranDev/week-3-solutions/blob/master/mandatory/2-bush-berries.js